install-dependencies.sh

```diff
@@ -129,11 +129,12 @@ fedora_python3_packages=(
     python3-pyudev
 )
 
-pip_packages=(
-    scylla-driver
-    geomet
-    traceback-with-variables
-    scylla-api-client
+# an associative array from packages to constrains
+declare -A pip_packages=(
+    [scylla-driver]=
+    [geomet]="<0.3,>=0.1"
+    [traceback-with-variables]=
+    [scylla-api-client]=
```

Comment:
It is here for a good reason, the `pip_packages` array is used for two different purposes in two different places:
1. It is used to set up the build system - for this we need the constrains since the original code, installs a version with a constrain (which is suspect was the reason that the installation commands were hardcoded in the firstplace)
2. It is used by the packaging scripts to extract the actual packages in order to extract them back from the building environment into the `scylla-python3` relocatable. In this case only the packages names is needed since
we are extracting the exact same version that was installed on the build machine.
Take a look in the additional change that prints the packages and you will notice that I print only the "keys" of the array.
BTW: I started as you suggested but then the when testing, the build failed, this is how I found out about the second case in the first place.